specification/graphrbac/data-plane/readme.typescript.md

```diff
@@ -7,7 +7,7 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 typescript:
   azure-arm: true
   package-name: "@azure/graph"
-  output-folder: "$(typescript-sdks-folder)/packages/@azure/graph"
+  output-folder: "$(typescript-sdks-folder)/sdk/graph/graph"
```

Comment:
Did you intentionally call this one graph instead of graphrbac? I think graph is probably the best name for the service directory but I wanted to at least call it out to make sure it was an intentional difference.

Reply:
So the team wanted `graphrbac` as the package name, Kamil said. But we don't want to change any package names for now. Also I did notice that other languages have the package name for this as graphrbac. For now I kept the service folder name as `graph` since it uses Azure Graph service. We can update it later if we want to.